packages/datadog-plugin-fetch/test/index.spec.js

```diff
@@ -125,6 +125,23 @@ describe('Plugin', () => {
         })
       })
 
+      it('should return the response', done => {
+        const app = express()
+        app.get('/user', (req, res) => {
+          res.status(200).send()
+        })
+        getPort().then(port => {
+          appListener = server(app, port, () => {
+            fetch(new globalThis.Request(`http://localhost:${port}/user`))
```

Comment:
Might be a good idea to use string/URL form here to be sure we don't break support for that, especially given _most_ people would likely be doing that rather than using a `Request` object. 🤔

Reply:
Not sure I understand. Both of these are used in other tests.

Reply:
Ah, sorry, ignore me. Missed it was used in the other tests. 😅